### DISCUSSION TITLE
so-user-update

### DISCUSSION THREAD

**Discussion by gcsrellis** [Security-Onion-Solutions/securityonion#2008] [Unsupported Versions]
I see this command listed but it doesnt actually work.  The email account login created during setup doesnt work when logging into the hive.  Its a complexity issue I discovered during setup. Works in all but playbook and the hive. I seen this command listed in hopes to change the password via cli. What is the current recommended way to change it without web gui.

**Comment by cm-ops** [comment]
Try `so-user update username@example.com`.

**Reply by gcsrellis** [reply]
That worked!  I assumed the syntax was the same as the others when tabbed out. Thanks for the help once again.
